### ISSUE TITLE
Telelphone Link is Clickable on Non-Mobile

### ISSUE THREAD

**Issue by MLKendall** [b2io/base2.io#329]
The telephone link is clickable even on non-mobile devices.  It should only be clickable on devices that allow phone calls.

**Comment by mewdriller** [comment]
It looks like there may be no viable way to accurately check it the browser supports `tel` and/or `sms`. 

https://stackoverflow.com/a/6933017

**Comment by tborres** [comment]
I don't believe that we should limit this to mobile devices anyway. Computers also support phone calls if a phone app has been specified by the user.

**Comment by mewdriller** [comment]
Sounds fair to me. If they have support, it will launch their preferred app. If not, it will prompt them to suggest a protocol handler.
